geopy/geocoders/base.py

```diff
@@ -185,6 +187,10 @@ def __init__(
                             else options.default_ssl_context)
 
         if self.proxies:
+            if isinstance(self.proxies, str):
+                self.scheme = self.proxies.split('://')[0]
```

Comment:
First of all, `proxies` shouldn't affect `self.scheme` in any way.

Proxy scheme and URL scheme are totally different and doesn't need to match.

For example, for `proxies = 'http://my_proxy_server:8080'` and `self.scheme = 'https'` the client would send a [CONNECT][1] method to the proxy (in plaintext), and then start TLS negotiation with the remote using that TCP connection.

Just remove this line and it will be fine.

[1]: https://tools.ietf.org/html/rfc7231#section-4.3.6